### PR TITLE
add minutes for March 22 2023 meeting

### PR DESCRIPTION
**Pull request by Trott**
I don't see the recording at https://www.youtube.com/@nodejs-foundation/streams. Maybe we didn't stream today's meeting? I left the recording blank in the minutes. Please update to indicate either the URL of the recording or a few words to say it wasn't streamed. Then pull it out of draft mode. Thanks!

### CODE REVIEW

**Comment by targos** [comment]
The stream is here: https://www.youtube.com/watch?v=jM8D4QqEmwI (on another YouTube channel @mcollina)

**Comment by Trott** [comment]
> The stream is here: https://www.youtube.com/watch?v=jM8D4QqEmwI (on another YouTube channel @mcollina)

Thanks. Added.

File: meetings/2023-03-22.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Which version of V8 will be supported for the longest time? What version should we use for Node v20 release? Targos to give recommendations.
+
+SEA:
+Joyee has sent a PR for the layout for <https://github.com/nodejs/node/pull/47125>
+Matteo: why do we need a layout?
+Joyee: it’s need to support snapshots.
+Darshan: also pre-baked CLI argos
+
+ShadowRealm:
+stacktrace decoration is coming <https://github.com/nodejs/node/pull/47107>.
```
Comment (tniessen) [inline meetings/2023-03-22.md:80]:
```suggestion
V8 Currency:
* V8 11.2 is in beta in Chrome and there is a PR open <https://github.com/nodejs/node/pull/46815>
* Workaround for compiling in ARM64 Windows
* Issues with WebAssembly / ARM, @targos to investigate today
* Almost ready to land
* Not sure we will be updating to the next version
* Which version of V8 will be supported for the longest time? What version should we use for Node v20 release? Targos to give recommendations.

SEA:
* Joyee has sent a PR for the layout for <https://github.com/nodejs/node/pull/47125>
* Matteo: why do we need a layout?
* Joyee: it’s need to support snapshots.
* Darshan: also pre-baked CLI argos

ShadowRealm:
* stacktrace decoration is coming <https://github.com/nodejs/node/pull/47107>.
```

**Review by legendecas — APPROVED** [review]

**Comment by mcollina** [comment]
I'm sorry about the wrong streaming location. I don't understand why it picked the wrong account.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by cjihrig — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by RaisinTen — APPROVED** [review]

**Pull request merged**